Library/Homebrew/test/test_dependency.rb

```diff
@@ -39,6 +39,13 @@ def test_accepts_symbol_and_string_tags
     assert_equal [:build, "bar"], dep.tags
   end
 
+  def test_merge_repeats
+    dep = Dependency.new("foo", [:build, "bar"], nil, "foo")
+    dep2 = Dependency.new("foo", [:build, "bar"], nil, "foo2")
```

Comment:
What is the benefit of setting the tags to `[:build, "bar"]` when the simpler `[]` also works?